### PR TITLE
[KIECLOUD-569] - Route name is not exposed in operator CR

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

File: pkg/controller/kieapp/defaults/defaults.go [hunk 1]
```diff
@@ -1956,3 +1972,48 @@ func setDefaultCors(cors *api.CORSFiltersObject) {
 	cors.MaxAgeName = "Access-Control-Max-Age"
 	cors.MaxAgeValue = Pint32(1)
 }
+
+func getRouteHostname(obj interface{}) (host string) {
+
+	switch o := obj.(type) {
+	case *api.KieServerSet:
+		if len(o.RouteHostname) > 1 {
```
Comment (vaibhavjainwiz) [inline pkg/controller/kieapp/defaults/defaults.go:1980]: Why its > 1? not > 0

Comment (spolti) [reply pkg/controller/kieapp/defaults/defaults.go]: no reason, I did it wrong :)
thanks.

**Review by jakubschwan — APPROVED** [review]
Lgtm

**Pull request merged**